File: python/gemini/p94.py
import bcrypt

def register_user(username, password):
    # Generate a salt and hash the password
    salt = bcrypt.gensalt()
    hashed_password = bcrypt.hashpw(password.encode('utf-8'), salt)
    
    # In a real app, you'd pass these to your database handler
    print(f"Username: {username}")
    print(f"Hashed Password: {hashed_password.decode('utf-8')}")

register_user("tech_enthusiast", "secure_password123")

File: java/gemini/p94.java
import org.springframework.security.crypto.bcrypt.BCrypt;

public class UserAuth {
    public static void main(String[] args) {
        String username = "java_admin";
        String password = "password_alpha";

        // Generate salt and hash
        String salt = BCrypt.gensalt();
        String hashedPassword = BCrypt.hashpw(password, salt);

        System.out.println("Username: " + username);
        System.out.println("Hashed Password: " + hashedPassword);
    }
}